### PR TITLE
First pass as React 0.14 migration for react-dnd

### PR DESCRIPTION
**Pull request by globexdesigns**
For those users that want to play around with react-dnd and React 0.14-rc1. This change should be all that's needed.

Also fixes https://github.com/gaearon/react-dnd/issues/252

### CODE REVIEW

**Comment by cookednoodles** [comment]
:+1:

File: site/components/CodeBlock.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import React, { PropTypes, Component, findDOMNode } from 'react';
+import React, { PropTypes, Component } from 'react';
+import {findDOMNode} from 'react-dom';
```
Comment (gaearon) [inline site/components/CodeBlock.js]: Style (spacing): should be `{ findDOMNode }`, not `{findDOMNode}`

File: src/bindConnectorMethod.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import shallowEqual from './utils/shallowEqual';
 import cloneWithRef from './utils/cloneWithRef';
 import { Disposable, SerialDisposable } from 'disposables';
-import { findDOMNode, isValidElement } from 'react';
+import { isValidElement } from 'react';
+import {findDOMNode} from 'react-dom';
```
Comment (gaearon) [inline src/bindConnectorMethod.js]: Style (spacing): should be `{ findDOMNode }`, not `{findDOMNode}`

File: package.json [hunk 3]
```diff
@@ -43,7 +43,8 @@
     "lodash": "^3.1.0"
   },
   "peerDependencies": {
-    "react": ">=0.13.0 <0.15.0 || ^0.14.0-alpha"
+    "react": ">=0.14.0 || ^0.14.0-rc1",
+	"react-dom": ">=0.14.0 || ^0.14.0-rc1"
```
Comment (gaearon) [inline package.json]: Style (spacing): extra indentation here

**Comment by gaearon** [comment]
Either way I can't merge this until React 0.14 is out because this cuts off 0.13 as valid peer.

**Comment by mattgi** [comment]
+1 now that 0.14 is out.

**Comment by gaearon** [comment]
+1s don't help.
No need to bump them. :-)

I'll work on this soon. Please ignore the warnings for now.

**Comment by gaearon** [comment]
Superseded by https://github.com/gaearon/react-dnd/pull/304, fixed in 2.0.0.
Release notes: https://github.com/gaearon/react-dnd/releases/tag/v2.0.0.

**Pull request closed without merging**